import "maidsafe/transport/transport.proto";

package maidsafe.kademlia.protobuf;

message Contact {
  required bytes node_id = 1;
  required transport.protobuf.Endpoint endpoint = 2;
  repeated bytes local_ips = 3;
  optional int32 local_port = 4;
  optional transport.protobuf.Endpoint rendezvous = 5;
}

message KadConfig {
  optional int32 port = 1;
  repeated Contact contact = 2;
}
